pkg/oci/utils.go

```diff
@@ -170,6 +170,13 @@ func PodConfig(runtime RuntimeConfig, bundlePath, cid, console string) (*vc.PodC
 		Console:     console,
 	}
 
+	if len(ocispec.Process.User.AdditionalGids) > 0 {
+		cmd.SupplementaryGroups = []string{}
```

Comment:
you can do simpler here, right ? Just loop over AdditionalGids array. And define cmd.SupplementaryGroups before so that you append only if there is actually some additional groups.